composer.json

```
@@ -27,6 +27,7 @@
         "drupal/extra_field": "^2.3",
         "drupal/facets_form": "^1.0.0-alpha6",
         "drupal/field_group": "^3.4",
+        "drupal/paragraphs": "1.17",
```

```suggestion
        "drupal/paragraphs": "~1.17.0",
```
we need to lock it in such a way that it still permits patch releases